packages/stateful/voting-module-adapter/adapters/DaoVotingCw4/actions/makeManageMembersAction/ManageMembersComponent.tsx

```diff
@@ -75,8 +76,16 @@ export const ManageMembersComponent: ActionComponent<
       title={t('title.manageMembers')}
     >
       <div className="flex flex-col gap-1">
-        <InputLabel name={t('form.membersToAddOrUpdate')} />
-        <div className="flex flex-col items-stretch gap-1">
+        <div className="flex flex-row gap-2">
+          <InputLabel name={t('form.membersToAddOrUpdate')} />
+          <TooltipInfoIcon
+            arrow
+            placement="top"
```

Comment:
nit: leaving placement empty will position it wherever it will best fit, so best to leave that out. also `arrow` is true by default. apologies for the lack of docs.

Reply:
i actually always want placement top because i don't want the tooltip to cover the members.

Reply:
does that sound OK to you?

Reply:
ohh look me at me assumin. nice.

Reply:
also, not all problems are docs. sometimes i just need to learn how to read ;)